app/routes/project/posts/post.js

```diff
@@ -25,6 +25,7 @@ export default Ember.Route.extend({
   },
 
   setupController(controller, models) {
+    this._super(controller, models);
```

Comment:
`setupController` is generally a hook where we might not want to call `_super`. The reason for that is that, by default, `setupController` looks like

```Javascript
setupController(controller, models) {
  return controller.set('model', models);
}
```

It doesn't really hurt anything, but it does add a "model" property which we aren't using to the controller.